Added a section dedicated to the rules label (#586)
akeneo/pim-helpcenter: content/md/productivity/get-started-with-the-rules-engine.md

The rules are defined in a YML file that you can import in the PIM, using a dedicated job profile.   

This feature is very, like, really powerful, and the use cases are countless. In order to guide you in the rules engine usage, we wrote this article where you will find out what you can use it for and how it works.

# Name your rules

> A rule is defined by a code (required) and you can name it with a label as well. This label can have a value per locale.

Reviewer: ```suggestion
Each rule must have a unique code. It can also have a label per enabled PIM locale. This label will appear under `Settings`/`Rules` and be also visible in your product form under your smart attributes.
```

> Here is the expected format:

Reviewer: ```suggestion
The labels settings must be added at the end of your rule, after the actions part. 
For instance, if you want to add a label for your en_US and fr_FR locales, here is the expected format:
```

Author: I guess the label settings can be put before or after, it doesn't matter (I just tested to put it first, before the priority and it seems to work, but @nmarniesse do you confirm?)

Reviewer 2: Yes there is no order for the keys `priority`, `conditions`, `actions` or `labels`. You can put them in the order you want.  
In yaml the order is important only for lists (when you have a `-` in front of the line)

Reviewer: Thanks @nmarniesse , I've edited my suggestion @morganen :)

```YML
  camera_copy_name_to_model:
        priority: 0
        conditions:
            - field: family
              operator: IN
              value:
                - camcorders
            - field: camera_model_name
              operator: EMPTY
        actions:
            - type: copy
              from_field: name
              to_field: camera_model_name
        labels:
            en_US: 'Copy name to model'
            fr_FR: 'Copie nom vers modèle'
```

> If no label is defined, we use the code between `[ ]` or `{}`.

Reviewer 2: `or {}` ? I don't know that. Where do we display that?

Reviewer: it's in your YML rule export file, both `[ ]` or `{}` can be used for the YML import

Author: On ne parle pas de la même chose ;) moi je parlais du fait que, comme partout dans le PIM, quand il n'y a pas de label de défini, on affiche le code entre crochets. Mais c'est peut être inutile de l'écrire.

Author: Du coup j'ai mal écrit ma phrase, je me suis mélangée les pinceaux suite à notre conversation Laure ;)

> If no label is defined, we use the code between `[ ]` or `{}`.

Reviewer: If you do not want to add labels to your rule, leave the labels settings with `[ ]` or `{ }`.

```YML
labels: {  }
```
 or 

```YML
labels: [  ]
```

:::warning
If no label is set, the rule's code will appear into brackets `[ ]` in the PIM user interface.
:::

Author: Seen with Pierre, we won't document the [] as it's an unexpected behavior from Php.

> 

Reviewer: ```suggestion

You will also be able to search on your rule labels under `Settings`/`Rules`.

```
